.github/workflows/test.yml

```diff
@@ -23,3 +23,26 @@ jobs:
 
       - name: Run tests
         run: tox -e ${{matrix.python-version}} -- -vrsx --color=yes
+
+  build-windows:
+    name: Test Windows build
+    runs-on: windows-latest
+    strategy:
+      max-parallel: 100
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - name: Checkout repos
+      uses: actions/checkout@v1
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Build x86_64 packages
```

Comment:
I think it's possible to run/test 32 bits package too: I'll try that in the grid

Reply:
Right, I've just added x86 packages as well.